generators/client/templates/webapp/app/locale/TranslationService.vue

```diff
@@ -22,7 +21,7 @@
                 if (this.$i18n && !this.$i18n.messages[this.currentLanguage]) {
                     let vm = this;
                     this.$i18n.setLocaleMessage(this.currentLanguage, {});
-                    axios.get(SERVER_API_URL + 'i18n/' + this.currentLanguage + '.json').then(function (res) {
+                    axios.get('i18n/' + this.currentLanguage + '.json').then(function (res) {
```

Comment:
Not sure how we do it angular/react by personally I like using template strings if possible.

Reply:
I think he is talking about the syntax, doing `` `i18n/${this.currentLanguage}.json` `` instead of string concat, and I agree, lets stick to es7/8 as much as possible, and since we will move to TS we can use all latest ES features

Reply:
Ok, sounds good indeed :smile:

Reply:
Totally agree, let's do the all the replacements in an other PR then.

Reply:
Do we agree on this ? Should we merge ?